docs/developer/eventing/sources/README.md

```diff
@@ -35,6 +35,7 @@ All Sources are part of the `sources` category.
 | [GitHub](../../../eventing/samples/github-source/README.md)            | v1alpha1 | Knative    | Registers for events of the specified types on the specified GitHub organization or repository, and brings those events into Knative. The GitHubSource fires a new event for selected [GitHub event types](https://developer.github.com/v3/activity/events/types/). See the [GitHub Source](../../../eventing/samples/github-source/README.md) example for more details.  |
 | [GitLab](../../../eventing/samples/gitlab-source/README.md) | v1alpha1 | Knative    | Registers for events of the specified types on the specified GitLab repository, and brings those events into Knative. The GitLabSource creates a webhooks for specified [event types](https://docs.gitlab.com/ee/user/project/integrations/webhooks.html#events), listens for incoming events, and passes them to a consumer. See the [GitLab Source](../../../eventing/samples/gitlab-source/README.md) example for more details.  |
 | [Heartbeats](https://github.com/knative/eventing/tree/main/cmd/heartbeats) |  N/A | Knative    | Uses an in-memory timer to produce events at the specified interval. |
+| [KogitoSource](https://github.com/knative-sandbox/eventing-kogito) |  v1 | Knative    | An implementation of the [Kogito Runtime](https://docs.jboss.org/kogito/release/latest/html_single/#proc-kogito-deploying-on-kubernetes_kogito-deploying-on-openshift) custom resource managed by the [Kogito Operator](https://github.com/kiegroup/kogito-operator). |
```

Comment:
I believe kogitosource is still `v1alpha1`
https://github.com/knative-sandbox/eventing-kogito/blob/main/config/300-kogitosource.yaml#L32

Reply:
Thank you, I figured it probably was but I wasn't sure so I just put v1 as a placeholder